test/io/ferguson/json-list2.chpl

```diff
@@ -0,0 +1,212 @@
+record MyRecord {
+  var numbers:list(int); // could it be [1..0] int ?
+}
+
+var myEntry:MyRecord;
+myEntry.numbers.push_back(1);
+myEntry.numbers.push_back(2);
+myEntry.numbers.push_back(3);
+
+writef("testing json write: %jt\n", myEntry);
+
+var f = opentmp();
+{
+  var writer = f.writer();
+  var str = '{"numbers": []}';
+  writeln("Writing JSON: ", str);
+  writer.write(str);
+  writer.close();
+}
+
```

Comment:
I tend to prefer simplicity in the tests over having less code. I don't think that these tests will be a maintenance burden.